### PR TITLE
#7 allow to ignore packages

### PR DESCRIPTION
**Pull request by zoechi**
- allow to ignore packages (don't include in output)
- prevent exception on invalid arguments
- add IntelliJ IDEA IDE workspace files to .gitignore

### CODE REVIEW

File: bin/pubviz.dart [hunk 1]
```diff
@@ -44,23 +56,23 @@ void _printUsage(ArgParser parser) {
   print('If <package path> is omitted, the current directory is used.');
 }
 
-String _getContent(VizRoot root, String format) {
+String _getContent(VizRoot root, String format, List<String> ignorePackages) {
```
Comment (kevmoo) [inline bin/pubviz.dart]: Make this a set

**Comment by zoechi** [comment]
I got a mail notification about a comment to change List<> to Set<> but I can't find the comment. 
Using a set is basically a good idea in this situation. 
The problem is that it's not possible to use an empty set as default value for optional parameters. 
Therefore I'm not sure it's a good idea but if you still prefer it I change it of course.

**Comment by kevmoo** [comment]
I'm tweaking your pull request locally. I want to add tests first. There are some issues with default values in named params. Stay tuned.

**Comment by kevmoo** [comment]
Thanks! Landed with tweaks in f80f7a409e833a7e6a2a31fa53bff0f3d5d5d259

**Pull request closed without merging**